dkist/dataset/tiled_dataset.py

```diff
@@ -190,3 +193,24 @@ def __repr__(self):
 
     def __str__(self):
         return dataset_info_str(self)
+
+    @property
+    def files(self):
+        """
+        A `~.FileManager` helper for interacting with the files backing the data in this ``Dataset``.
+        """
+        return self._file_manager
+
+    @property
+    def _file_manager(self):
+        return FileManager(
+            StripedExternalArray(
```

Comment:
I think here we need some more safeguards that all the tiles have the same settings in their respective file managers. target, dtype, shape, basepath, and I think even chunksize should all be pulled from the `FileManager` instance of the first tile and then verified that all the tiles are the same otherwise this should error.